borgcron.sh

```diff
@@ -256,10 +256,14 @@ getInfoAboutLastBackup() {
 
 # GUI functions (can be overwritten in config file)
 guiCanShowNotifications() {
-	# uses a command to get out, whether we can show notifications
-	# You can manually set this to return "false" (or 1, which is shell-speak for false)
-	# to disable all notifications.
-	command -v zenity >/dev/null
+	if [ -z "$DISPLAY" ]; then
```

Comment:
Please add a comment here explaining that this does exclude headless-servers etc.